src/jit/compiler.cpp

```diff
@@ -4713,165 +4713,165 @@ unsigned getMethodBodyChecksum(__in_z char *code, int size)
 }
 
 
-int           Compiler::compCompileHelper (CORINFO_MODULE_HANDLE            classPtr,
-                                           COMP_HANDLE                      compHnd,
-                                           CORINFO_METHOD_INFO            * methodInfo,
-                                           void *                         * methodCodePtr,
-                                           ULONG                          * methodCodeSize,
-                                           CORJIT_FLAGS                   * compileFlags,
-                                           CorInfoInstantiationVerification instVerInfo)
-    {
-        CORINFO_METHOD_HANDLE methodHnd = info.compMethodHnd;
+int           Compiler::compCompileHelper(CORINFO_MODULE_HANDLE            classPtr,
+                                          COMP_HANDLE                      compHnd,
+                                          CORINFO_METHOD_INFO            * methodInfo,
+                                          void *                         * methodCodePtr,
+                                          ULONG                          * methodCodeSize,
+                                          CORJIT_FLAGS                   * compileFlags,
+                                          CorInfoInstantiationVerification instVerInfo)
+{
+    CORINFO_METHOD_HANDLE methodHnd = info.compMethodHnd;
```

Comment:
If you find it necessary to reformat this whole method, you should also bring it up to coding convention standards. Otherwise, I think that you should not change the formatting.